qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -183,12 +179,41 @@ def main(self):
                     from collections import deque
                     print("".join(deque(open(f), MAX_LINES_TO_PRINT)))
         if success:
-            print("Tests successful")
+            self.log.info("Tests successful")
             sys.exit(0)
         else:
-            print("Failed")
+            self.log.warning("Failed")
             sys.exit(1)
 
+    def start_logging(self):
+        # Add logger and logging handlers
+        self.log = logging.getLogger('TestFramework')
+        self.log.setLevel(logging.DEBUG)
```

Comment:
I still don't agree. logging at non-standard levels is harder (you need to call `self.log.log(lvl, message`) rather than `self.log.debug(message)` or similar). Why would anyone want do that, and why would we want to allow people to add confusing logging levels to the test cases?